src/components/ProgressIndicator.vue

```diff
@@ -150,21 +150,24 @@ export default class ProgressIndicator extends Vue {
 
   get progressProps() {
     return {
-      class: ['text-like-cyan', 'fill-like-cyan'],
+      class: 'text-like-cyan fill-like-cyan',
       x: -this.measurement.height,
       width: this.progressWidth,
       height: this.measurement.height,
       rx: this.rx,
     };
   }
 
+  get progressContainerClasses() {
+    return [
+      'text-like-cyan-light',
+      { 'text-shade-gray': this.preset === ProgressIndicatorPreset.Thin },
+      'fill-like-cyan-light',
+    ];
+  }
+
   get progressContainer() {
     return {
-      class: [
-        'text-like-cyan-light',
-        { 'text-shade-gray': this.preset === ProgressIndicatorPreset.Thin },
-        'fill-like-cyan-light',
-      ],
```

Comment:
Using an object with key `class` is not flattened with `v-bind`, so I move the class array to separated computed prop